build/toolchain/wasm/BUILD.gn

```diff
@@ -32,4 +35,56 @@ gcc_toolchain("wasm") {
   is_clang = true
 
   link_outputs = [ "{{root_out_dir}}/{{target_output_name}}.wasm" ]
+
+  extra_toolchain_args = {
+    skia_use_angle = false
```

Comment:
These can be hooked up to GN args from `args.gn` by introducing new arguments with `declare_args` like `skia_use_angle_for_wasm` (or a better name).

Reply:
Do you mean that we'd have a file called `args.gn` over in flutter/engine that enumerates all the properties below, and it would override the same properties listed in https://github.com/google/skia/blob/main/modules/canvaskit/canvaskit.gni?

Reply:
No, sorry. I need to think about this PR some more. I'm going to spend some time with it this morning, and see if I can make some better suggestions.

Reply:
It seems all these attributes are destructured into some if/else block in the skia_wasm_lib used to build canvaskit: https://github.com/google/skia/blob/main/modules/canvaskit/BUILD.gn#L96

Can we convert the Skia rule into a template that takes these same attributes as `invoker.ATTR`?

Reply:
Yes. I think that would be a better way to go here.